class/class7.1/src/UnitTests.hpp

```diff
@@ -0,0 +1,119 @@
+#pragma once
+
+#include <vector>
+#include <functional>
+#include <string>
+#include <iostream>
+#include <stdexcept>
+#include <source_location>
+
+
+class Rectangle {
```

Comment:
looks strange to have those class declarations in unit tests header. better to move them into separate one